src/odemis/util/test/util_test.py

```diff
@@ -144,6 +145,32 @@ def test_simple(self):
             o = util.almost_equal(*i)
             self.assertEqual(o, eo, "Failed to get correct output for %s" % (i,))
 
+
+class PerpendicularDistanceTestCase(unittest.TestCase):
+
```

Comment:
doc string what that test does

Reply:
As the name itself implies that it's a test case for "PerpendicularDistance", and I don't think there is much to add to that, I didn't write a docstring.